BLSQ/iaso
# IA-2387: Completeness stats infos

Have a message telling you to select a form to be able to see the completeness results on a map

Related JIRA tickets : IA-2387

## Self proofreading checklist


## Changes

Create generic component to display an info tooltip nearby one input and use it

## How to test

Open completeness stats page and check that you have one infos icon nearby the form input.
Also double check if the org unit type dialog is still working, has i create a component for those two features

## Print screen / video

![Screenshot 2023-09-04 at 11 09 00](https://github.com/BLSQ/iaso/assets/12494624/c265861d-a24e-45e4-a6c1-154aead12d12)
